frontend/tests/acceptance/step_definitions/addNewJobApplicationContext.js

```diff
@@ -0,0 +1,75 @@
+const { I } = inject();
+const newJob = require('../pages/addNewJobApplication');
+const axios = require('axios');
+const loginPage = require('../pages/loginPage');
+const dashboard = require('../pages/dashboardPage');
+const {users} = require('../Globals');
+
+const apiUrl = process.env.API_SERVER_URL || 'http://localhost:3000';
+
+const ELEMENT =  newJob.elements;
+
+Given('user has registered with following data:', async (table) => {
+    const newUser = table.parse().hashes()[0];
+    users.push({ email: newUser.email, password: newUser.password });
+    try{
+        await axios.post(`${apiUrl}/auth/register`, newUser)
+        .then(({data}) => {
+            // users.push({ email: newUser.email, password: newUser.password });
+            console.log(data.message)
```

Comment:
are we just doing the console.log the message here!!

Reply:
it will log `user created` message